src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
I don't care much what the browser would do. One may want to generate inline source maps and external maps at the same time, and there's no reason for one of these options to cause another option to not take effect. If we really did want to make these options exclusive, we would error out immediately and tell the user to choose only one of them.